modules/viz/CMakeLists.txt

```diff
@@ -35,7 +34,14 @@ ocv_add_accuracy_tests()
 ocv_add_perf_tests()
 ocv_add_samples(opencv_imgproc opencv_calib3d opencv_features2d opencv_flann)
 
-ocv_target_link_libraries(${the_module} PRIVATE ${VTK_LIBRARIES})
+
+if (VTK_VERSION VERSION_LESS "8.90.0")
```

Comment:
Lets make condition obvious?
- `if(COMMAND vtk_module_autoinit)`
- or `if(DEFINED VTK_USE_FILE)`

Reply:
@alalek , this condition is used in official VTK samples (e.g. https://lorensen.github.io/VTKExamples/site/Cxx/SimpleOperations/DistanceBetweenPoints/#cmakeliststxt).